src/web/core/src/test/java/org/geoserver/web/data/resource/ResourceConfigurationPageTest.java

```diff
@@ -448,12 +448,12 @@ public void testWFSDataStoreResource() throws IOException {
         // verify Layer`s resource is updated with metadata
         assertNotNull(layerInfo.getResource().getMetadata().get(FeatureTypeInfo.OTHER_SRS));
 
-        // click first item in SRS (4326)
+        // click first item in SRS (urn:ogc:def:crs:EPSG::4326)
```

Comment:
Can you actually check the values? "urn:ogc:def:crs:EPSG::4326" shows in the comments, but it's not actually verified.

Reply:
understood, will do.